modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -61,10 +59,8 @@ logrotate /etc/logrotate.conf
 # Change the access level of configs/ rules/ and run_forseti.sh
 chmod -R ug+rwx ${forseti_home}/configs ${forseti_home}/rules ${forseti_home}/install/gcp/scripts/run_forseti.sh
 
-# Install tracing libraries
-pip install .[tracing]
```

Comment:
Why did we drop this?

Reply:
I don't know why was that there in the first place at all. That line fails causing the whole script to fail. forseti-security DM script doesn't have `pip install .[tracing]`

Reply:
Ohh, got it. Good catch!